src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -237,10 +239,130 @@ private static void PromptRequiredProperties<T>(T manifest, VersionManifest vers
+                {
+                    Console.Clear();
+                    Console.WriteLine();
+                    Logger.InfoLocalized(nameof(Resources.DisplayPreviewOfSelectedInstaller_Message));
+                    var serializer = Serialization.CreateSerializer();
+                    string installerString = serializer.Serialize(installerManifest);
+                    Console.WriteLine(installerString);
+                    Console.WriteLine();
+                }
+                else
+                {
+                    Installer selectedInstaller = MatchMenuSelectionToInstaller(selectedItem, installerManifest);
+                    PromptHelper.DisplayPropertiesAsMenuSelection(selectedInstaller, Resources.None_MenuItem);
+                }
+            }
+        }
+
+        private static Installer MatchMenuSelectionToInstaller(string selectedItem, InstallerManifest installerManifest)
+        {
+            string[] selection = selectedItem.Split('|');
+            var matchedInstallers = installerManifest.Installers.Where(
+                item => item.Architecture == (InstallerArchitecture)Enum.Parse(typeof(InstallerArchitecture), selection[0]) &&
+                item.InstallerType == (InstallerType)Enum.Parse(typeof(InstallerType), selection[1], true) &&
+                item.InstallerUrl == selection[2]);
+
+            Installer selectedInstaller;
+            if (matchedInstallers.Count() > 1)
+            {
+                Scope scope;
+                Scope? nullableScope = null;
+                string installerLocale = string.Empty;
+
+                for (int i = 3; i < selection.Length; i++)
+                {
+                    // if parsing for scope enum fails, value must be an installer locale
+                    if (Enum.TryParse(selection[i], true, out scope))
+                    {
+                        nullableScope = scope;
+                    }
+                    else
+                    {
+                        installerLocale = selection[i];
+                    }
+                }
+
+                selectedInstaller = matchedInstallers.Single(
+                    item => nullableScope != null && item.Scope == nullableScope &&
+                    !string.IsNullOrEmpty(installerLocale) && item.InstallerLocale == installerLocale);
+            }
+            else
+            {
+                selectedInstaller = matchedInstallers.Single();
+            }
+
+            return selectedInstaller;
+        }
+
+        private static List<string> GenerateInstallerSelectionList(List<Installer> installers)
+        {
+            List<string> selectionList = new List<string>();
+            selectionList.Add(Resources.AllInstallers_MenuItem);
+
+            foreach (Installer installer in installers)
+            {
+                var installerTuple = string.Join('|', installer.Architecture, installer.InstallerType.ToEnumAttributeValue(), installer.InstallerUrl);

```

Comment:
IMO this would look better with a space on either side of the pipe, " | "

Reply:
I agree, changed to suggestion